client/src/assets/images/icons/index.js

```diff
@@ -25,7 +25,7 @@ export const CohortIcon = () => (
 );
 
 export const ListIcon = () => (
-  <Icon data-icon="clipboard-list" viewBox="0 0 384 512">
+  <Icon aria-label="list" viewBox="0 0 384 512">
```

Comment:
Please investigate https://stackoverflow.com/questions/22039910/what-is-aria-label-and-how-should-i-use-it